example/README: update to reflect Go 1.14+ requirement (#260)
gopherjs/vecty: example/README.md

Vecty fully supports two major Go <-> Web compilers thanks to its minimal dependencies:

- Go 1.14+ WebAssembly support
- [GopherJS](https://github.com/gopherjs/gopherjs) (Go to JavaScript transpiler)

If you are just getting started, we suggest using the Go 1.14+ WebAssembly support.

## Go 1.14+ WebAssembly support

> Vecty requires Go 1.14+ as it makes use of [synchronous callback support](https://go-review.googlesource.com/c/go/+/142004) which is not present in earlier versions of Go. **Ensure you are running Go 1.14 or higher.**

Reviewer: ```suggestion
**Ensure you are running Go 1.14 or higher.** Vecty requires Go 1.14+ as it makes use of improvements to the `syscall/js` package which is not present in earlier versions of Go.
```
